### DISCUSSION TITLE
[求助]大学生毕业没工作怎么办

### DISCUSSION THREAD

**Discussion by cljstxy19960229** [share121/inter-knot#5460] [委托]
救命啊大學生畢業已經沒有就業方向惹

**Comment by JolinJostar** [comment]
遇事不决公务员🥹

**Comment by xingyun-kb** [comment]
找工作不要急
天命集团欢迎你

**Reply by buroro** [reply]
天命加班苦竞争大，不如来我逆熵<img src="https://img-static.mihoyo.com/communityweb/upload/6d59035ac87518e06d32fb3342657e2a.gif" alt="给你两拳" width="75" >

**Reply by aomengxin** [reply]
小时工作制，灵活上下班.ipg

**Comment by kunga555** [comment]
要不要试试地铁安检？虽然很累，但是坐地铁免费

**Comment by xintie2023** [comment]
私立学校保安，包吃包住。
